src/geocat/viz/util.py

```diff
@@ -179,6 +179,82 @@ def add_right_hand_axis(ax,
     return axRHS
 
 
+def add_height_from_pressure_axis(ax,
+                                  heights=None,
+                                  pressure_units='hPa',
+                                  ticklabelsize=12,
+                                  label='Height (km)',
+                                  labelpad=10,
+                                  axislabelsize=16):
+    """Utility function that adds a right-hand Height axis to the plot, derived
+    from the left-hand Pressure axis.
+
+    Parameters
+    ----------
+
+        ax (:class:`matplotlib.axes._subplots.AxesSubplot` or :class:`cartopy.mpl.geoaxes.GeoAxesSubplot`):
+            Current axes to the current figure
+
+        heights (:class:`numpy.ndarray` or list):
+            An array of desired height values in km.
+
+        pressure_units (:class:`str`):
+            The Pint-compliant unit string associated with the Pressure values. Assume to be hPa.
+
+        ticklabelsize (:class:`int`):
+            Text font size of tick labels. A default value of 12 is used if nothing is set.
+
+        label (:class:`str`):
+            Text to use for the right hand side label.
+
+        labelpad (:class:`float`):
+            Spacing in points from the axes bounding box. A default value of 10 is used if nothing is set.
+
+        axislabelsize (:class:`int`):
+            Text font size for y-axes. A default value of 16 is used if nothing is set.
+
+        Returns
+        -------
+
+        axRHS (:class:`matplotlib.axes._subplots.AxesSubplot` or :class:`cartopy.mpl.geoaxes.GeoAxesSubplot`):
+            The created right-hand axis
+    """
```

Comment:
The docstring here is still giving sphinx some issues. The current preview looks like this: 
<img width="808" alt="image" src="https://user-images.githubusercontent.com/38434768/234130320-c845687b-6c39-4a0a-a5fd-7285ef140153.png">

It actually looks like multiple docstrings from the package are giving sphinx trouble, I'd maybe look at docstrings in comp. For this docstring, I think the main issue is the additional indent starting immediately under the Parameters header.

Reply:
Ahh probably the multiple ones that I copied the template from! I found different methods for formatting it within the module. After the build, I'll see if this function looks better and if so, fix all of the others.

Reply:
It looks better now! I'll fix the other doc strings and contributor's guide in a separate PR.